### ISSUE TITLE
glfw errors when building on Mac

### ISSUE THREAD

**Issue by galvinben** [liamg/aminal#26]
**Describe the bug**
When trying to build on Mac I receive this output:

```
# github.com/liamg/aminal/vendor/github.com/go-gl/glfw/v3.3/glfw
ld: framework not found MoltenVK
clang: error: linker command failed with exit code 1 (use -v to see invocation)
# github.com/liamg/aminal/vendor/github.com/go-gl/glfw/v3.3/glfw
native_darwin.c:7:17: warning: implicit declaration of function 'glfwGetCocoaWindow' is invalid in C99 [-Wimplicit-function-declaration]
native_darwin.c:7:9: warning: cast to 'void *' from smaller integer type 'int' [-Wint-to-void-pointer-cast]
native_darwin.c:11:17: warning: implicit declaration of function 'glfwGetNSGLContext' is invalid in C99 [-Wimplicit-function-declaration]
native_darwin.c:11:9: warning: cast to 'void *' from smaller integer type 'int' [-Wint-to-void-pointer-cast]
```

**To Reproduce**
1. `go build` on macOS Mojave (10.14)
2. See error

**Expected behavior**
Should build successfully with no errors.

**Environment (please complete the following information):**
 - macOS Mojave 10.14
 - go version go1.11 darwin/amd64

**Comment by galvinben** [comment]
Also tried installing glfw3, but still errors.

**Comment by liamg** [comment]
Fixed in #27
